### ISSUE TITLE
Properties that are marked as readOnly are not read only

### ISSUE THREAD

**Issue by ItsReddi** [cebe/yii2-openapi#4]
The generated code does not respect the "readOnly" property.
This is because of the generated model rules. and the requestbody is loaded against the DatabaseModel.

It could be a good idea to create a validator for each endpoint to be able to mix the same property on different endpoints.

**Comment by cebe** [comment]
yeah, validation should use the schema definition instead of yii model rules, this is currently not implemented.
